src/sssom/constants.py

```diff
@@ -37,14 +37,10 @@
 ]
 
 UNKNOWN_IRI = "http://w3id.org/sssom/unknown_prefix/"
-PREFIX_MAP_MODE_METADATA_ONLY = "metadata_only"
-PREFIX_MAP_MODE_SSSOM_DEFAULT_ONLY = "sssom_default_only"
-PREFIX_MAP_MODE_MERGED = "merged"
-PREFIX_MAP_MODES = [
-    PREFIX_MAP_MODE_METADATA_ONLY,
-    PREFIX_MAP_MODE_SSSOM_DEFAULT_ONLY,
-    PREFIX_MAP_MODE_MERGED,
-]
+MergeMode = Literal["metadata_only", "sssom_default_only", "merged"]
+PREFIX_MAP_MODE_METADATA_ONLY: MergeMode = "metadata_only"
```

Comment:
Never seen this syntax. What does this do, when considering line 40? Looks redundant at first glance (twice the same assignment)

Reply:
This makes checking that one of the constants is used properly is checked by type checking. It further builds the string enumeration into the automatic documentation generation